tox.ini

```diff
@@ -39,9 +39,8 @@ deps =
     coverage
 
     django-formtools
-    django-phonenumber-field>=0.7.2,<0.99
+    django-phonenumber-field>=1.1.0,<2.99
     django_otp>=0.3.4,<0.99
-    phonenumbers>=7.0.9,<7.99
```

Comment:
`phonenumbers` is a direct dependency, see also: https://github.com/Bouke/django-two-factor-auth/blob/0ec439604abebfd087f3bb282ec8cc2333777b29/two_factor/templatetags/two_factor.py#L3

Reply:
Ah. I have updated `setup.py` to include the dep then, rather than dropping it from `tox.ini`.